zio-http-gen/src/test/resources/ComponentOrderWithAliases.scala

```diff
@@ -1,12 +1,14 @@
 package test.component
 
 import zio.schema._
+import zio.schema.annotation.validate
+import zio.schema.validation.Validation
 
 case class Order(
   id: OrderId.Type,
   product: String,
-  @zio.schema.annotation.validate[Int](zio.schema.validation.Validation.greaterThan(0)) quantity: Int,
-  @zio.schema.annotation.validate[Double](zio.schema.validation.Validation.greaterThan(-1.0)) price: Double,
+  @validate[Int](Validation.greaterThan(0)) quantity: Int,
+  @validate[Double](Validation.greaterThan(-1.0)) price: Double,
```

Comment:
Not related to current PR, but the float/double validation logic is faulty.
`minimum 0` does not mean `> -1.0`
Since `-0.5 > -1.0`, but does not adhere to original minimum requirement.

CC @987Nabil

Reply:
True. But I think this is unsolvable then. There is no way to translate it to zio-schema correctly right now. We probably need to add new validations to zio schema

Reply:
@987Nabil in second thought, perhaps you need a change for greaterOrEqualTo, but we can still use `java.lang.Math.nextDown` (or `nextAfter`) to get the closest representable floating point number that is smaller (or larger) than the input.

It might be ugly, e.g:
```sacla
@ java.lang.Math.nextDown(0f)
res0: Float = -1.4E-45F
```
but at least it'll be correct.

WDYT?

Reply:
I think that works work. But I would not write out the number, but generate the expression

Reply:
Yup...
```scala
@validate(Validation.greaterThan(Math.nextDown(0f)))
```
Would be much better than (and understandable) than generating weird epsilons